.eslintrc

```diff
@@ -13,15 +13,16 @@
   ],
   "rules": {
     "@typescript-eslint/lines-between-class-members": "off",
+    "@typescript-eslint/naming-convention": "off",
     "no-unused-expressions": "error",
     "class-methods-use-this": "off",
     "consistent-return": "off",
     "arrow-body-style": "off",
     "padded-blocks": "off",
+    "array-callback-return": "off",
     "no-console": "warn",
     "import/prefer-default-export": "off",
     "quotes": ["error", "single"],
-    "indent": ["error", 2],
```

Comment:
was there no quick fix for standardising the indendations?

Reply:
We are using typescript indenting rule not the eslint indent rule so they are not conflicting